http/_io.ts

```diff
@@ -237,11 +237,8 @@ export async function writeResponse(
   const protoMajor = 1;
   const protoMinor = 1;
   const statusCode = r.status || 200;
-  const statusText = STATUS_TEXT.get(statusCode);
+  const statusText = r.statusText ?? STATUS_TEXT.get(statusCode) ?? "";
```

Comment:
An empty status text might confuse some HTTP parsers so maybe don't allow that. The PR looks fine otherwise.

Reply:
So maybe something like below would be better?
```diff
if (!statusText) {
-  throw new Deno.errors.InvalidData("Bad status code");
+  throw new Deno.errors.InvalidData("Missing status text");
}
```

Reply:
Yes, I'd say so.  Allowing `{ statusCode: 777, statusText: "" }` (statusText deliberately empty but not omitted) is something I'd be supportive of, too. That way people can deliberately opt into it if they wanted to.